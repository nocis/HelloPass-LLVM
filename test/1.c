
void test(int a, int b, int c, int d, int e, int f) {
	c = a + b;
	d = a * c;
	e = a;
	a = a + 1;
	d = a + b;
	f = e + b;
	f = e * f;
}

int main() {
	test(1,2,3,4,5,6);
	return 0;
}
