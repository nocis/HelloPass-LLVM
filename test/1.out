3=1+2
4=1*3
6=1+5
7=6+2
3=1+2
4=1*3
